assets/src/application/ui/input/InlineEditInput/InlineEditHeading.tsx

```diff
@@ -4,8 +4,8 @@ import Editable from './Editable';
 import { HeadingProps } from './types';
 
 const InlineEditHeading: React.FC<HeadingProps> = ({ ellipsis, level = 4, ...rest }) => {
-	const enableEllipsis = ellipsis ? ellipsis : { rows: 1, expandable: false };
-	return <Editable inputType='heading' ellipsis={enableEllipsis} level={level} {...rest} />;
+	const ellipsisOptions = ellipsis ? ellipsis : { rows: 2, expandable: false };
+	return <Editable ellipsis={ellipsisOptions} level={level} {...rest} inputType='heading' />;
```

Comment:
why `inputType='heading'` is at the end and `{...rest}` in the middle?

Reply:
meant to write a comment to explain that but forgot... :\

the inputType prop has strict values and this order prevents it from being overwritten, not that that would be likely to happen, but basically just means that `InlineEditHeading` will **always** use `inputType='heading'`